minicluster/src/test/java/org/apache/accumulo/minicluster/MiniAccumuloClusterTest.java

```diff
@@ -98,7 +100,8 @@ public void checkDFSConstants() {
   }
 
   @SuppressWarnings("deprecation")
-  @Test(timeout = 30000)
+  @Test
+  @Timeout(value = 30000, unit = TimeUnit.MILLISECONDS)
```

Comment:
This comment applies throughout this module. We only used millis because we had to. They should all be seconds now that we don't need to use millis.

```suggestion
  @Timeout(value = 30, unit = TimeUnit.SECONDS)
```